components/packaging/common/load-telemetry-plugin

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env sh
+
+BASE=/var/lib/docker
```

Comment:
We can get this from the API.

I think a separate rpm/deb might be the way to go. It can have it's own systemd unit file that depends on the docker service being started and not make any assumptions about the running docker daemon.